ajax/libs/Captionator/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Captionator",
+  "filename": "js/captionator-min.js",
+  "description": "Simple closed-captioning polyfill for HTML5. Just 8KB when gzipped!",
+  "version": "0.5",
+  "homepage": "http://captionatorjs.com/",
+  "author": {
+    "name": "Christopher Giffard",
+    "url": "https://github.com/cgiffard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cgiffard/Captionator.git"
+  },
+  "keywords": [
+    "Captionator",
+    "captionator.js",
+    "js"
+  ],
+  "license": "BSD-2-Clause",
```

Comment:
How do I know it's BSD-2-Clause? I didn't see any related keywords in the project, if you use the some license detector to match its license in readme with all the well know licenses, I'll respectfully ask you to confirm that the author wouldn't mind the project to be marked as "BSD-2-Clause"